masq_lib/src/test_utils/mock_websockets_server.rs

```diff
@@ -192,17 +192,19 @@ impl MockWebSocketsServer {
                                 log(
                                     do_log,
                                     index,
-                                    "Responding to a request for FireAndForget message in dirrection to UI",
+                                    "Responding to a request for FireAndForget message in direction to UI",
                                 );
-                                inner_responses_arc
-                                    .lock()
-                                    .unwrap()
-                                    .clone()
-                                    .into_iter()
-                                    .for_each(|message| {
-                                        client.send_message(&message).unwrap();
-                                        thread::sleep(Duration::from_millis(20))
-                                    })
+                                let (ordinal_number, sender) =
```

Comment:
This is far too complicated. There has to be a simpler way. One simpler way might be to put at least the `ordinal_number` in the `broadcastTrigger` message. I'm not sure how to get the `sender` in there, but sending it over a global variable is probably not the right way. Perhaps an `Option` property on `MockWebsocketsServer` that can be set after the server is created?